modules/apps/wysiwyg/wysiwyg-web/bnd.bnd

```diff
@@ -0,0 +1,6 @@
+Bundle-Name: Liferay Wysiwyg Web Conversion
+Bundle-SymbolicName: com.liferay.wysiwyg.web
+Bundle-Version: 1.0.0
+Liferay-Releng-Module-Group-Description:
+Liferay-Releng-Module-Group-Title:
+Web-ContextPath: /wysiwyg-web
```

Comment:
No, it should not be in Journal since we would not be able to release it.

It should be, as you were creating as an independent App which will be released as a new version of the existing WYSIWYG portlet App.